rust/feature-flags/src/flags/flag_operations.rs

```diff
@@ -81,21 +81,7 @@ impl FeatureFlagList {
             FlagError::DatabaseUnavailable
         })?;
 
-        let query = r#"
-            SELECT f.id,
-                  f.team_id,
-                  f.name,
-                  f.key,
-                  f.filters,
-                  f.deleted,
-                  f.active,
-                  f.ensure_experience_continuity
-              FROM posthog_featureflag AS f
-              JOIN posthog_team AS t ON (f.team_id = t.id)
-            WHERE t.project_id = $1
-              AND f.deleted = false
-        "#;
-
+        let query = "SELECT id, team_id, name, key, filters, deleted, active, ensure_experience_continuity, version FROM posthog_featureflag WHERE team_id = $1 AND deleted = false";
```

Comment:
don't do this; this should be a query that uses the `project_id`, and not the team ID.  It's annoying that the SQL didn't complain but this needs to be that join query.

Reply:
Ah, bad merge on my part. Good catch!